src/components/molecules/Header.tsx

```diff
@@ -1,38 +1,46 @@
-import React, { FC } from 'react';
+import React, { FC, useEffect } from 'react';
 import { observer } from 'mobx-react-lite';
 import { AppBar, Logo } from '../atoms';
 import AnywayImage from '../../assets/anyway.png';
 import { SignInIcon } from '../atoms/SignInIcon';
 import LogInLinkGoogle from './LogInLinkGoogle';
 import { Box } from '@material-ui/core';
+import { useStore } from '../../store/storeConfig';
+import RootStore from '../../store/root.store';
+import UserProfileHeader from './UserProfileHeader';
 import { makeStyles } from '@material-ui/core/styles';
 import LanguageMenu from '../organisms/LanguageMenu';
 
 const useStyles = makeStyles({
   settings: {
     display: 'flex',
+    width: '40%',
```

Comment:
?

Reply:
maybe it's a wrong name for this class- it represents the user which already logged into the app, it is 40% of the width of the entire header, I will change the name `userSection`